pkgs/development/compilers/gcc/8/default.nix

```diff
@@ -121,6 +121,7 @@ stdenv.mkDerivation ({
              grep -q _DYNAMIC_LINKER "$header" || continue
              echo "  fixing \`$header'..."
              sed -i "$header" \
+  inherit version;
```

Comment:
It seems like there are bunch of problems in this commit now...

Reply:
Oops!